handlers/api.go

```diff
@@ -4095,12 +4095,12 @@ func adjustQueryResults(data []interface{}, adjustQueryEntriesFuncs ...func(map[
 	return nil
 }
 
-// SendErrorResponse exposes sendErrorResponse
-func SendErrorResponse(w http.ResponseWriter, route, message string) {
-	sendErrorResponse(w, route, message)
+// SendBadRequestResponse exposes sendErrorResponse
+func SendBadRequestResponse(w http.ResponseWriter, route, message string) {
+	sendBadRequestResponse(w, route, message)
 }
```

Comment:
**Question:** Why not remove `sendBadRequestResponse` and only have `SendBadRequestResponse`? I know, it was like this before, but I am asking as you have touched it.
IF you want to change it, you should also take a look at https://github.com/gobitfly/eth2-beaconchain-explorer/blob/77cf0bef9cdc9d872321f989479eca66e4ce334c/handlers/api.go#L4123-L4126 for consistency.

Reply:
fix